### PR TITLE
LPS-70064 SF

### PR DESCRIPTION
**Pull request by leoadb**

### CODE REVIEW

**Comment by leoadb** [comment]
ci:retest

**Comment by vicnate5** [comment]
@leoadb The rest of the forms tests are turned off at the moment.

The upgrades tests are unable to save a form.

Javascript error is thrown after user goes to the Form builder the first time
```
JAVA_SCRIPT_ERROR: Error: Form template renderer is not defined: "ddm.tabbed_form [http://localhost:8080/combo/?browserId=firefox&minifierType=&languageId=en_US&b=7004&t=1504712327035&/o/dynamic-data-mapping-form-renderer/js/form_pagination_support.js&/o/dynamic-data-mapping-form-renderer/js/form_tabs_support.js&/o/dynamic-data-mapping-form-renderer/js/form_template_support.js&/o/dynamic-data-mapping-form-renderer/js/type.js&/o/dynamic-data-mapping-form-renderer/js/expressions_evaluator.js&/o/dynamic-data-mapping-form-renderer/js/form_validation_support.js&/o/dynamic-data-mapping-form-renderer/form.js&/o/dynamic-data-mapping-form-renderer/js/soy_template_util.js&/o/frontend-js-web/aui/datatype-date-parse/datatype-date-parse-min.js&/o/frontend-js-web/aui/datatype-date-format/lang/datatype-date-format_en-US.js&/o/frontend-js-web/aui/datatype-date-format/datatype-date-format-min.js&/o/frontend-js-web/aui/datatype-date-math/datatype-date-math-min.js&/o/frontend-js-web/aui/datatype-number-parse/datatype-number-parse-min.js&/o/frontend-js-web/aui/datatype-number-format/datatype-number-format-min.js:11]
```

**Comment by leoadb** [comment]
ci:retest

**Comment by leoadb** [comment]
ci:retest

**Comment by vicnate5** [comment]
I manually canceled the https://github.com/liferay/com-liferay-dynamic-data-mapping/pull/389#issuecomment-327547120 retest. The second one is running.

**Comment by leoadb** [comment]
@vicnate5 thanks!

**Comment by leoadb** [comment]
@brunobasto can you help us with the Javascript error Victor mentioned?

File: dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -278,6 +278,7 @@
 
 				&.option-selected {
 					color: #65B3F0;
+					font-weight: bold;
```
Comment (leoadb) [inline dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/css/main.scss]: @rafaprax Please do not merge this one

**Comment by vicnate5** [comment]
@leoadb Forms still not saving :(

**Comment by leoadb** [comment]
ci:retest

**Comment by leoadb** [comment]
ci:stop

**Comment by leoadb** [comment]
ci:retest

**Comment by leoadb** [comment]
ci:retest

**Comment by leoadb** [comment]
@vicnate5 I couldn't reproduce locally the errors of saving a Form. Can you please try to reproduce those? The error of captcha usage is a real one. I'm looking into that right now. Thanks!

**Comment by leoadb** [comment]
ci:retest

**Comment by leoadb** [comment]
ci:stop

**Comment by leoadb** [comment]
ci:retest

**Comment by leoadb** [comment]
ci:stop

**Comment by leoadb** [comment]
ci:retest

**Comment by vicnate5** [comment]
@leoadb Sidebar fails to load again here.

**Comment by leoadb** [comment]
ci:retest

**Comment by vicnate5** [comment]
@leoadb I can reproduce the sidebar not loading locally.

What I did:
1. Empty DDM directory in master
2. Copy DDM subrepo files (this pull) to master
3. Build master
4. Start portal with QA testing portal-ext
5. Go to Forms > Add > Add Field > Text

Sidebar does not load

**Comment by leoadb** [comment]
ci:retest

**Comment by vicnate5** [comment]
@leoadb The form sidebar issue is fixed. The form is still sometimes failing to save.

**Comment by leoadb** [comment]
@vicnate5 Yeah. It's seems the fix I sent did not work. I'm looking into it.

**Comment by rafaprax** [comment]
ci:retest

**Pull request merged**